### DISCUSSION TITLE
Should "credited as" be filled if it is identical to the entity name?

### DISCUSSION THREAD

**Discussion by kellnerd** [kellnerd/musicbrainz-scripts#14] [General]
Continuing the discussion from https://github.com/ROpdebee/mb-userscripts/issues/77#issuecomment-1005379935

> @tigerman325: One issue that some editors might not like is that it fills the "credited as" field with a label name even if it's the exact same. Usually that would be left blank.

Fair point, the primary reason why I decided to fill the credit in all cases is that I don't know which name the user-selected entity will have at that point and it would require additional logic to clear the credit again if it is identical. I think this should be done by MBS itself if that is the desired usage, otherwise I don't think it harms (especially if it's automated and has not to be entered by hand)?

Always filling the "credited as" field also helps to keep the exact credit if the label entity should ever be renamed (which would cause the relationship to show up under the new name if it has no credit).

**Comment by tigerman325** [comment]
I actually prefer to keep them there as well, but I used to do that and was discouraged. Not sure why, as you put it, if the labels change this keeps them how they are on the release.
